lib/src/util/logger.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:convert' show JsonEncoder;
```

Comment:
We should probably think about tidying up our logger/logger wrappers to promote reuse

Reply:
Can we discuss this one over a call later?

Reply:
sure; it's not urgent, nor even particularly important :-) We can put it on the "eventually" list